docs/index.md

```diff
@@ -1,13 +1,17 @@
-# What is CQtDeployer
-The CQtDeployer is deployment tool application for extract all depends library of executable and create launch script for your projects.
+# CQtDeployer
+
+Here you can find documentation for the CQtDeployer in different languages. It describes:
+
+* Information on how to use this deployment Tool. 
+
+* Main description (description of the functionality and capabilities of the program)
+
+* links to installers 
```

Comment:
CQtDeployer create installer using Qt Installer Framework. But documentation do not have a links to installers.
Fix it please